webpack/assets/javascripts/react_app/routes/Models/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ModelsPage from './ModelsPage';
+
+export default {
```

Comment:
this is a great start, I guess we would like to add more routes here in the future (new ,delete, edit)